lib/InternalBytecode/03-ES6Class.js

```diff
@@ -0,0 +1,61 @@
+/* @nolint */ var initES6Class = function () {
+  function defineClass(ctor, superClass) {
+    ctor.prototype = Object.create(superClass && superClass.prototype, {
```

Comment:
We generally recommend caching global object lookups as local variables on first run, so you can rely on them to not be overridden by the user. As a bonus, it'll be faster to call functions that are already local variables instead of having to do a lookup on the global object.

```
// inside initES6Class
var Object_create = Object.create;

// inside defineClass
ctor.prototype = Object_create(/* args */);
```

Similarly for `Object.defineProperty`, etc.